python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -512,6 +529,19 @@ def parseAndRun(cls, args=None, config=None, log=None, doReturnResults=False):
 
         taskRunner = cls.RunnerClass(TaskClass=cls, parsedCmd=parsedCmd, doReturnResults=doReturnResults)
         resultList = taskRunner.run(parsedCmd)
+
+        try:
+            nFailed = sum(((res.exitStatus != 0) for res in resultList))
+        except Exception as e:
+            parsedCmd.log.warn("Unable to retrieve exit status (%s); assuming success" % e)
```

Comment:
Probably not important in this context, but in general you shouldn't use `%` to expand the string but separate with commas:

    parsedCmd.log.warn("Unable to retrieve exit status (%s); assuming success", e)

(Another below).